Doc/library/profile.rst

```diff
@@ -121,6 +121,14 @@ results to a file by specifying a filename to the :func:`run` function::
 The :class:`pstats.Stats` class reads profile results from a file and formats
 them in various ways.
 
+You can also do inline profiling with :class:`Profile`::
```

Comment:
```suggestion
You can also do inline profiling with :class:`~cProfile.Profile`::
```

Reply:
Using `cProfile.Profile` makes sense, because the context manager only works with `cProfile`. However, I don't think the `~` should be used.